docs/public-networks/get-started/connect/sync-node.md

```diff
@@ -47,6 +47,14 @@ You can store the world state using [Forest of Tries](../../concepts/data-storag
 
 You can run a full node using [snap synchronization (snap sync)](#snap-synchronization), [checkpoint synchronization (checkpoint sync)](#checkpoint-synchronization), or [fast synchronization (fast sync)](#fast-synchronization).
 
+:::note Sync nodes for BFT
+
+If you're running a node on a [QBFT](../../../private-networks/how-to/configure/consensus/qbft.md) or [IBFT 2.0](../../../private-networks/how-to/configure/consensus/ibft.md) network, your node must use [fast sync](#fast-synchronization) or [full sync](#run-an-archive-node). 
+
+Full sync is set by default.
```

Comment:
```suggestion
Snap sync and checkpoint sync are not supported for [QBFT](../../../private-networks/how-to/configure/consensus/qbft.md) or [IBFT 2.0](../../../private-networks/how-to/configure/consensus/ibft.md) networks. 
```

As suggested in my previous comments on the previous PR, we should call out snap sync and checkpoint sync here to make it relevant to the location of the note.